conf/modules.config

```
@@ -433,14 +433,20 @@ process {
     }
 
     withName: 'OPENMS_TEXTEXPORTER' {
+        publishDir  = [
+            enabled: false
+        ]
+    }
+
+    withName: 'PREPARE_FOR_MULTIQC' {
         publishDir  = [
             path: {"${params.outdir}"},
             mode: params.publish_dir_mode,
-            pattern: '*.tsv'
+            pattern: '*_final.tsv'
```

No `final`needed right?
```suggestion
            pattern: '*.tsv'
```